lib/pliny/error_reporter.rb

```diff
@@ -14,29 +13,35 @@ def self.notify(exception, context: {}, rack_env: {})
     end
   end
 
-  class RollbarReporter
-    include ::Rollbar::ExceptionReporter
-    include ::Rollbar::RequestDataExtractor
+  begin
```

Comment:
+1. The best I can up with is moving the reporter to its own file as suggested below, and then just load and configure it from an initializer:

``` ruby
require 'lib/pliny/error_reporters/rollbar'
Pliny::ErrorReporters << Pliny::RollbarLogger.new

Rollbar.configure do |config|
  config.enabled = ENV.has_key?('ROLLBAR_ACCESS_TOKEN')
...
```

(Or something.)